demos/interactive_face_detection_demo/face.hpp

```diff
@@ -24,22 +19,17 @@ struct Face {
     void updateAge(float value);
     void updateGender(float value);
     void updateEmotions(std::map<std::string, float> values);
-    void updateHeadPose(HeadPoseDetection::Results values);
+    void updateHeadPose(HeadPoseResults values);
     void updateLandmarks(std::vector<float> values);
 
     int getAge();
     bool isMale();
     std::map<std::string, float> getEmotions();
     std::pair<std::string, float> getMainEmotion();
-    HeadPoseDetection::Results getHeadPose();
+    HeadPoseResults getHeadPose();
     const std::vector<float>& getLandmarks();
     size_t getId();
 
-    void ageGenderEnable(bool value);
-    void emotionsEnable(bool value);
-    void headPoseEnable(bool value);
-    void landmarksEnable(bool value);
-
     bool isAgeGenderEnabled();
     bool isEmotionsEnabled();
     bool isHeadPoseEnabled();
```

Comment:
Please remove these methods (and the underlying flags) from here.
Refactor the rest of the code appropriately.

Reply:
Removed methods
Changed the code, see previous comment